data/rulesets.js

```diff
@@ -374,6 +387,7 @@ exports.BattleFormats = {
 	abilityclause: {
```

Comment:
Should possibly be named Hackmons Ability Clause.

Reply:
AAA uses it too

Reply:
Two Ability Clause?

Reply:
Yeah, AAA and BH and both use the same ability clause.

Reply:
No, I mean, how about naming it "Two Ability Clause"?

Reply:
@TheImmortal I'd say this is up to you

Reply:
Ability Clause is fine imo.

Reply:
I soft-veto "Ability Clause". Pick a better name.